### DISCUSSION TITLE
Webpack 5 build fails with error: configuration[0].module.rules[3].issuer has an unknown property 'test'

### DISCUSSION THREAD

**Discussion by hitendra-ap** [vercel/next.js#25041] [Help]
### What version of Next.js are you using?

10.2.0

### What version of Node.js are you using?

12

### What browser are you using?

chrome

### What operating system are you using?

macOS

### How are you deploying your application?

next build

### Describe the Bug

After adding future flag for webpack 5 in next.config, build is failing with Error:"configuration[0].module.rules[3].issuer has an unknown property 'test'". However `module` configuration to webpack is being provided by the next.js inetrnally. i tried to print the rule itself where the error comes in:
```
{
    test: /\.(jpe?g|png|svg|gif|ico|webp|jp2)$/,
    issuer: { test: /\.\w+(?<!(s?c|sa)ss)$/i },
    exclude: undefined,
    use: [ [Object] ]
  }
```

### Expected Behavior

Build should go fine after adding webpack 5 flag

### To Reproduce

Enable webpack 5 flag with latest nextjs version.

**Comment by hitendra-ap** [comment]
Hi, Any suggestions, How can I fix it locally?. I believe this rule has been added internally by nextjs.

**Reply by timneutkens** [reply]
It's not added by Next.js, likely something in your project but no reproduction was provided.

**Reply by hitendra-ap** [reply]
Thanks for your help, I found it's fixed when I upgrade `next-images` module
